src/basic.rs

```diff
@@ -81,6 +90,11 @@ pub type BasicTokenResponse = StandardTokenResponse<EmptyExtraTokenFields, Basic
 pub type BasicTokenIntrospectionResponse =
     StandardTokenIntrospectionResponse<EmptyExtraTokenFields, BasicTokenType>;
 
+///
+/// Basic OAuth2 revocable token type
+///
+pub type BasicRevocableToken = StandardRevocableToken;
```

Comment:
I agree with your reasoning, but note that one consequence of removing it will be that the definition of BasicClient will no longer consistently use BasicXxx types, it would then be:

```
pub type BasicClient = Client<
    StandardRevocableToken,
    BasicErrorResponse,
    BasicTokenResponse,
    BasicTokenType,
    BasicTokenIntrospectionResponse,
    BasicRevocationErrorResponse,
>;
```

Is that okay? It looks a bit strange, perhaps confusing?

Reply:
looks ok to me! I agree consistency is important, but I think having separate `BasicRevocableToken` and `StandardRevocableToken` types that are interchangeable can also lead to confusion since people can wonder "what's the difference between these? when should I use one over the other?"

🤷‍♂️ seems like both solutions have potential downsides